src/main/java/io/cdap/plugin/gcp/bigtable/source/HBaseColumn.java

```diff
@@ -0,0 +1,38 @@
+package io.cdap.plugin.gcp.bigtable.source;
+
+import java.util.regex.Pattern;
+
+/**
+ * This class represents a reference to HBase column
+ */
+public class HBaseColumn {
+  private static final Pattern PATTERN = Pattern.compile("__");
+
+  private final String family;
+  private final String qualifier;
+
+  private HBaseColumn(String family, String qualifier) {
+    this.family = family;
+    this.qualifier = qualifier;
+  }
+
+  public static HBaseColumn fromFullName(String fullName) {
+    String[] parts = PATTERN.split(fullName, 2);
+    String family = parts[0];
+    String qualifier = parts[1];
+    return new HBaseColumn(family, qualifier);
+  }
+
+  public static boolean isValidName(String fullName) {
+    String[] parts = PATTERN.split(fullName);
+    return 2 == parts.length;
```

Comment:
it is valid to have '__' in the qualifier, which will result in more than 2 parts. This should check that there are at least 2 parts.

Reply:
fixed